utility/src/test/java/org/oppia/util/data/DataProvidersTest.kt

```diff
@@ -57,6 +61,12 @@ private const val COMBINED_STR_VALUE_02 = "I used to be indecisive. At least I t
 @LooperMode(LooperMode.Mode.PAUSED)
 @Config(manifest = Config.NONE)
 class DataProvidersTest {
+  /**
+   * Returns Mockito.any() as nullable type to avoid java.lang.IllegalStateException when
+   * null is returned.
+   */
+  fun <T> any(): T = Mockito.any<T>()
```

Comment:
Can you create an issue mentioning that we need to move these functions into a helper class? You can assign that to me, and add a comment with the issue here in the following format:
`TODO(<issue number>): Add a MockitoHelper class to handle nullable versions of all mockito matchers`

Reply:
Done 
Issue created - #1233

Reply:
Few nits:
```
 // TODO(#1233): Add a MockitoHelper class to handle nullable versions of all mockito matchers.
  /**
   * Returns Mockito.any() as nullable type to avoid java.lang.IllegalStateException when
   * null is returned.
   */
  fun <T> any(): T = Mockito.any<T>()
```
1. The TODO should comment on top instead of bottom.
2. There should no space between `TODO` and `(`
3. Full stop (.) at the end of sentence.

Reply:
Done

Reply:
I think you don't need this any() function anymore

Reply:
Done